src/EditorFeatures/Core/Diagnostics/AbstractDiagnosticsTaggerProvider.SingleDiagnosticKindPullTaggerProvider.cs

```diff
@@ -94,10 +94,6 @@ protected sealed override ITaggerEventSource CreateEventSource(ITextView? textVi
             if (!_callback.IsEnabled)
                 return;
 
-            var diagnosticMode = GlobalOptions.GetDiagnosticMode();
-            if (!_callback.SupportsDiagnosticMode(diagnosticMode))
-                return;
```

Comment:
since this provider is only for inline-diagnostics (which are handled by roslyn in all cases) we don't need this check.